### DISCUSSION TITLE
Displaying my full Windows Path right from the drive

### DISCUSSION THREAD

**Discussion by Dark-Knight499** [JanDeDobbeleer/oh-my-posh#5847] [Troubleshoot]
minimal setup I need is C:\path\repo_name (branch_name)> read through the docs could not make it 
the how to show drive info ?

**Comment by JanDeDobbeleer** [comment]
@Dark-Knight499 the drive is part of the path, no? i don't see what's missing here, can you explain with a few examples?

**Reply by Dark-Knight499** [reply]
for example my path is C:\Users\user_name\folder\repo

When using any default theme, the terminal shows the path as folder\repo instead of the full path, which should start from the drive (i.e., C:\Users\user_name\folder\repo). Even when building a custom theme, I am unable to display the full path starting from the drive letter

I wanted to build a custom theme with just the path and the branch name 
e.g. C:\Users\user_name\folder\repo (main)

**Reply by JanDeDobbeleer** [reply]
@Dark-Knight499 the `path` segment has a `style` property and you're looking for `full`. See [here](https://ohmyposh.dev/docs/segments/system/path#style).

**Comment by marcstober** [comment]
I think the issue is that `C:\Users\user_name` is the home directory, and the home directory gets replaced with `~` or whatever is configured as the `home_icon`. I ran into this, too. It took a bit of work to get the same Windows path that non-posh PowerShell shows in all cases, that is, (1) when you're in the home directory itself, (2) when you're in a subfolder of the home directory, and (3) when you're anywhere else:

```json
"properties": {
    "style": "full",
    "folder_separator_icon": "\\",
    "home_icon": "{{ .Location }}"
},
"template": "{{ $loc := .Location | replace \"/\" \"\\\\\" }}{{ path $loc .Location }}<p:white>></>",
```

It would be nice if there was just an option to turn off the home directory substitution. What makes it particular complex is that when you are in the home directory itself, `.Location` seems to work a little differently than when you are in a subdirectory. 

One of my use cases is teaching beginning programmers, and I want what I to show is a highlighted but not entirely different version of non-posh PowerShell that my students would find on a less tricked out machine. E.g., for experienced people the forward slashes on Windows might be fine or even preferable, but I don't want to impose the cognitive load on a student of having to understand why they are getting backward slashes instead of forward slashes like I have. (And if they ask, "how did you get those colors?", _then_ I can introduce them to the world of OMP :) ).

**Reply by JanDeDobbeleer** [reply]
@marcstober you can always use `.Env.PWD` for this as that should be the unmodified path.
